packages/desktop-gui/src/project-nav/browsers_spec.jsx

```diff
@@ -59,7 +59,9 @@ describe('Browsers', () => {
     browserListOpened()
 
     cy.log('**closing**')
-    cy.get('.spacer').click()
+    // Click anywhere next to the dropdown to make sure it closes
+    cy.get('.spacer').click({ force: true })
+    browserListClosed()
```

Comment:
Added the assertion to make sure the browser list actually closed after clicking somewhere else